src/index.js

```diff
@@ -15,3 +15,13 @@ export function render(app, url, options = {}) {
   const ctx = renderContext(url, options);
   return simulate(app, ctx);
 }
+
+let describe = null;
+let it = null;
+
+if (typeof jest !== 'undefined') {
```

Comment:
It might be worth doing something fancy here so there's nice error messages if folks attempt to use this outside of a jest environment.

Perhaps getters/setters or Proxies?

Reply:
Added support for throwing an error, which I think makes sense for now?